src/cli/run_command.zig

```diff
@@ -1507,6 +1508,7 @@ pub const RunCommand = struct {
 
         // Run absolute/relative path
         if ((script_name_to_search.len > 1 and script_name_to_search[0] == '/') or
```

Comment:
don't we have a way to know if a path is absolute?
```suggestion
        if (bun.path.isAbsolute(script_name_to_search))
```

Reply:
i can make this change, but there is a difference between this code and isAbsolute, in that it is checking for a path that is not just the root dir.

this would make `bun run /` never run a script named `/`

why would you name a script this? no idea, but it would no longer be usable. through bun.